### DISCUSSION TITLE
Menu component interaction

### DISCUSSION THREAD

**Discussion by sang-yeo** [twilio-labs/paste#1076] [Q&A] [answered]
Can the Menu component show a list item when a user selects it? Like this? 
<img width="747" alt="Screen Shot 2021-01-06 at 2 30 13 PM" src="https://user-images.githubusercontent.com/67388401/103826967-c04d3300-502c-11eb-9d51-06b000e093dc.png">

**Comment by SiTaggart** [comment, marked as answer]
Hi @sang-yeo 

This is the key difference between a Menu and a Combobox.

A Menu is used to perform an action. It's trigger is static, it never changes its meaning. It's always the "bookmarks" menu, or the "more actions" menu. Upon performing the action, it's meaning never changes, it's still the "more actions" menu.

A Combobox is used to _select_ a value. As you are changing the text, you are in-fact selecting a value and subsequently displaying that selection to the user. So you should be using the Combobox here.

**Reply by sang-yeo** [reply]
Thanks for a quick response. I saw this design that's why I created above interaction. 
<img width="215" alt="Screen Shot 2021-01-06 at 2 48 52 PM" src="https://user-images.githubusercontent.com/67388401/103827736-854bff00-502e-11eb-9331-a7f263bfb18f.png">

**Reply by serifluous** [reply]
Ah yeah, that's a mistake in some in-progress work. That should definitely be a combobox! We'll get it updated.
